includes/apple-exporter/components/class-embed-generic.php

```diff
@@ -51,6 +51,7 @@ public static function node_matches( $node ) {
 			&& 'iframe' === $node->childNodes->item( 0 )->nodeName
 		) {
 			return $node;
+			/* phpcs:enable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase */
```

Comment:
it probably works fine as is, but I feel like the enable shouldn't be inside the `if`... my mind wants it to be at the same level as the disable.